.github/workflows/tekton-test.yml

```diff
@@ -154,7 +154,7 @@ jobs:
             - name: MACHINE_ID
               value: test
             - name: ENERGY_SOURCE
-              value: intel_rapl
+              value: trained_power_model
```

Comment:
isn't this wrong? doesn't it mean we are using estimator even in tekton workflow?

Reply:
It is intentionally. Tekton test on GH action runs on VM and its source is trained_power_model.